examples/async-immutable/store/configureStore.js

```diff
@@ -0,0 +1,23 @@
+import { createStore, applyMiddleware } from 'redux';
```

Comment:
Is this file from another example? And I don't think it's actually used anywhere, right?

Reply:
Yes, the code is originally from https://github.com/angular-redux/ng-redux/tree/master/examples/async. I think you are right. The file isn't used in this example.